2016/ideas-list-swc.md

```diff
@@ -82,33 +101,16 @@ Skills required to complete the project (you can apply without some of them):
 * Knowledge of [Git][] for storing your work on [GitHub][]
 * Knowledge of SurveyMonkey OAuth.
 
-Any of these skills could be learned during the project and I will help you
-with them, but you probably can't learn *all* of them during GSOC.
-
-### Mentors
-
-* [@pbanaszkiewicz](https://github.com/pbanaszkiewicz)
-  (timezone: UTC+1h [Poland])
-* [@rgaiasc](https://github.com/rgaiacs)
-  (timezone: UTC+0h [UK])
+Any of these skills could be learned during the project and we will help you
+with them, but you probably can't learn *all* of them during GSoC.
```

Comment:
`Skills required […]` first.
Then `Any of these skills […]`.
At the end (notice I changed it a little) `This project will be a single application, but you'll be designing and writing it from scratch.`.